src/gpio.rs

```diff
@@ -55,6 +56,15 @@ impl<'a> GpioDriver<'a> {
         }
     }
 
+    pub fn gpio_by_id(pinid:usize) -> TockResult<Gpio<'a>> {
```

Comment:
I'm not sure why Travis didn't run (it checks formatting), but I think there should be a space between the colon and `usize` here. I think the correct command to run to format the code is `cargo fmt`.

Reply:
You can run `./run_all_checks.sh` to verify your formatting. Travis is triggered only when bors receives a command.

Reply:
I fixed the formatting, I did not know about cargo fmt.